sources/packages/backend/apps/api/src/route-controllers/institution/_tests_/e2e/institution.aest.controller.createInstitution.e2e-spec.ts

```diff
@@ -0,0 +1,102 @@
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+  });
+
+  it("Should create institution when validate is passed.", async () => {
+    // Arrange
+    const payload = {
+      legalOperatingName: "Create Institution legal operating name",
+      operatingName: "Create Institution operating name",
+      institutionType: 1,
+      regulatingBody: "icbc",
+      establishedDate: "2023-06-01",
+      primaryEmail: "test@test.ca",
+      primaryPhone: "7785367878",
+      website: "https://www.test.ca",
+      primaryContactFirstName: "Primary",
+      primaryContactLastName: "Contact",
+      primaryContactEmail: "test@test.ca",
+      primaryContactPhone: "7785367878",
+      mailingAddress: {
+        addressLine1: "123 Gorge Rd E",
+        addressLine2: "",
+        selectedCountry: "Canada",
+        country: "Canada",
+        city: "Victoria",
+        postalCode: "V1V1V1",
+        provinceState: "BC",
+        canadaPostalCode: "V1V1V1",
+      },
+      mode: "create",
+      clientType: "aest",
+    };
+    const endpoint = "/aest/institution";
+    const token = await getAESTToken(AESTGroups.BusinessAdministrators);
+
+    // Act/Assert
+    let institutionId: number;
+    await request(app.getHttpServer())
+      .post(endpoint)
+      .send(payload)
+      .auth(token, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.CREATED)
+      .then((response) => {
+        expect(response.body.id).toBeGreaterThan(0);
+        institutionId = response.body.id;
+      });
+    const savedInstitution = await db.institution.findOne({
+      where: { id: institutionId },
+    });
+    expect(savedInstitution).toEqual(
+      expect.objectContaining({
+        businessGuid: null,
+        establishedDate: payload.establishedDate,
+        institutionAddress: {
+          mailingAddress: {
+            addressLine1: payload.mailingAddress.addressLine1,
+            addressLine2: payload.mailingAddress.addressLine2,
+            city: payload.mailingAddress.city,
+            country: payload.mailingAddress.country,
+            postalCode: payload.mailingAddress.postalCode,
+            provinceState: payload.mailingAddress.provinceState,
+            selectedCountry: payload.mailingAddress.selectedCountry,
+          },
+        },
+        institutionPrimaryContact: {
+          email: payload.primaryContactEmail,
+          firstName: payload.primaryContactFirstName,
+          lastName: payload.primaryContactLastName,
+          phone: payload.primaryContactPhone,
+        },
+        institutionTypeId: payload.institutionType,
+        legalOperatingName: payload.legalOperatingName,
+        operatingName: payload.operatingName,
+        otherRegulatingBody: null,
+        primaryEmail: payload.primaryEmail,
+        primaryPhone: payload.primaryPhone,
+        regulatingBody: payload.regulatingBody,
+        website: payload.website,
+      }),
+    );
+  });
+
+  afterAll(async () => {
```

Comment:
I believe that some other test files are missing this, can you please take a look?